server/puppeteerserver.js

```diff
@@ -0,0 +1,47 @@
+const puppeteer = require('puppeteer');
+const express = require('express');
+const cors = require('cors');
+const fs = require('fs');
+const app = express();
+
+app.use(cors());
+
+app.get('/getPDF', async (req, res) => {
+  createHTMLFile(req.query.invoicePreview);
+  const resPDF = await generatePDF();  
+  DeleteFile(`${__dirname}/invoice.html`);
```

Comment:
function names should not start with caps

Reply:
change function name as deleteFile